src/opensuse.ipxe

```diff
@@ -2,6 +2,60 @@
 
 # OpenSUSE Operating System
 # http://opensuse.org
+
+isset ${dhcp-server} || goto static_ip
+set netsetup netsetup=dhcp
+goto goto_menu
+
+:static_ip
+# Need to convert netmask into prefix, because otherwise the installer
```

Comment:
This could probably be moved into a common library or might even be useful in iPXE itself (if it isn't already there) but it's fine for now.